README.md

```diff
@@ -30,6 +31,121 @@ cd myAwesomeApp
 ns run ios|android
 ```
 
+## Update to v3 from v2
+
+### Start of the application
+As the way of starting a vue application, NativeScript-Vue has changed, has also done so to follow the vueE3 format.
```

Comment:
vueE3 likely a typo

Reply:
@vpiskunov thanks for reviewing it! I'll change it in a while!